app/src/common/shared/org/mozilla/vrbrowser/browser/PermissionDelegate.java

```diff
@@ -178,7 +178,12 @@ public void onContentPermissionRequest(GeckoSession aSession, String aUri, int a
         } else if (aType == PERMISSION_GEOLOCATION) {
             type = PermissionWidget.PermissionType.Location;
         } else if (aType == PERMISSION_MEDIA_KEY_SYSTEM_ACCESS) {
-            callback.grant();
+            if (SettingsStore.getInstance(mContext).isDrmContentPlaybackEnabled()) {
+                callback.grant();
+            } else {
+                callback.reject();
+            }
+            mWidgetManager.getFocusedWindow().setDrmUsed(true);
```

Comment:
Aren't we supposed to show a popup the first time asking the using if they want to enable EME? Right now I have to go into setting and turn it on in order for it to work.

Reply:
I see two options here:
- Show the DRM button as enabled/disabled every time a page request a DRM permission and show a notification ("DRM is disabled") in case it is globally disabled. This is more aligned with what we now (after the URL standardization) do for popups.
- Hide the DRM button it in case it is globally disabled and show a dialog.

Reply:
I think it should work as follows (just my opinion):
Show dialog on first permission request. Allow user to grant/deny global EME permission.
After that we either show the enabled or disabled icon in the url bar. If the user clicks the icon in the url bar they can either globally enable/disable from there. They can also toggle in settings menu.
No block list for first version. It's either on or off.

Reply:
I've implemented your last suggestion. I'm still waiting for the popup blocked icon though.